content/applications/marketing/marketing_automation/campaign_templates/double_optin.rst

```diff
@@ -0,0 +1,153 @@
+includes engaged contacts in the mailing list.
+
+The *Marketing Automation* app's *Double Opt-in* campaign template automatically configures a new
+campaign to send an email to new recipients to confirm their consent.
+
+When the *Double Opt-in* campaign template is used, a new mailing list titled *Confirmed contacts*
+is created in the *Email Marketing* app, and any new mailing list contacts that are added to the
+default *Newsletter* mailing list are sent a confirmation email to double opt-in. The contacts that
+click on a link in the email are automatically added to the *Confirmed contacts* mailing list.
+
+.. important::
+   When using the *Double Opt-in* campaign template, only the contacts in the *Confirmed contacts
+   mailing* list are considered to have confirmed their consent.
+
+.. _marketing_automation/template/using-double-optin:
+
+Use the Double Opt-in campaign template
+=======================================
+
+Open the :menuselection:`Marketing Automation` app, and select the :guilabel:`Double Opt-in`
+campaign template to create a new campaign for confirming consent.
+
+.. tip::
+   The campaign templates do not display by default when there are existing *Marketing Automation*
+   campaigns. To display the campaign templates, type the name of a campaign that does not exist in
+   the database into the :guilabel:`Search...` bar, then press :kbd:`Enter`.
+
+   For example, searching for `empty` displays the campaign template cards again, as long as there
+   is not a campaign with the name "empty" in the database.
+
+Campaign configuration
+----------------------
+
+Upon creation of the campaign, the campaign form loads with a new preconfigured campaign.
+
+The :guilabel:`Target` and :guilabel:`Filter` configuration of the campaign is as follows:
+
+- :guilabel:`Name`: `Double Opt-in`
+- :guilabel:`Responsible`: The user who created the campaign.
+- :guilabel:`Target`: :guilabel:`Mailing Contact`
+- :guilabel:`Unicity based on`: :guilabel:`Email (Mailing Contact)`
+- :guilabel:`Filter`:
+
+  - :guilabel:`Email` :guilabel:`is set`
+  - :guilabel:`Blacklist` :guilabel:`is not` :guilabel:`set`
+  - :guilabel:`Mailing lists` :guilabel:`contains` `Newsletter`
+
+.. important::
+   The :guilabel:`Target` model of the campaign should not be modified. Changing the
+   :guilabel:`Target` model with activities in the :guilabel:`Workflow` invalidates the existing
+   activities in the :guilabel:`Workflow`.
+
+   The *Double Opt-in* campaign template is intended to only use the :guilabel:`Mailing Contact`
+   model.
+
+The campaign loads two activities in the :guilabel:`Workflow` section of the campaign: an email
+activity with a child server action activity that triggers *on click*.
+
+By default, the `Confirmation` email activity is set to trigger :guilabel:`1 Hours` after the
+beginning of the workflow. In other words, the email is sent 1 hour after a new contact is added to
+the *Newsletter* mailing list.
```

Comment:
```suggestion
By default, the :guilabel:`Confirmation` email activity is set to trigger :guilabel:`0 Hours` after the
beginning of the workflow. In other words, the email is sent right after a new contact is added to
the *Newsletter* mailing list.
```
I'm suggesting the addition of a GUI because it actually *appears* on-screen, but I am actually seeing that it is set to '0' hours, by default -- but again, that may be a 'runbot' specific discrepancy. worth a double-check during your revisions either way.
![Screenshot 2024-05-29 110715](https://github.com/odoo/documentation/assets/73958186/738c8270-5a31-421d-a041-3aaeda56a112)